### ISSUE TITLE
Fixing malformed fuzzy origin wrapping features in GenBank

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#2133]
As of PR #1770 for issue #1745, the GenBank parser will attempt to fix simple malformed origin-wrapping features like ``6800..100`` on a 7000bp sequence, and change them to ``join(6800..7000,1..100)`` when the sequence is defined as circular. It would still give a warning about the location string.

However, this does not consider fuzzy locations like ``<6800..100`` or ``6800..>100`` etc as in the test case from @ybendana reported here:

https://github.com/biopython/biopython/issues/1608#issuecomment-492009046

Test case here:

https://github.com/biopython/biopython/files/3192665/SYNPUC19CV_cross_origin_feat.txt

```
LOCUS       SYNPUC19CV              2686 bp    DNA     circular SYN 17-MAY-2019
DEFINITION  Cloning vector pUC19c, complete sequence.
ACCESSION   L09137
VERSION     L09137.2
KEYWORDS    .
SOURCE      Cloning vector pUC19c
  ORGANISM  Cloning vector pUC19c
            other sequences; artificial sequences; vectors.
REFERENCE   1  (bases 1 to 2686
...
     misc_feature    <2644..159
                     /standard_name="cross origin trunc"
...
```

Currently this fails with an exception, e.g.

``` 
$ python3 -c "from Bio import SeqIO; print(SeqIO.read('SYNPUC19CV_cross_origin_feat.txt', 'gb').id)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/pc40583/repositories/biopython/Bio/SeqIO/__init__.py", line 726, in read
    first = next(iterator)
  File "/Users/pc40583/repositories/biopython/Bio/SeqIO/__init__.py", line 661, in parse
    for r in i:
  File "/Users/pc40583/repositories/biopython/Bio/GenBank/Scanner.py", line 489, in parse_records
    record = self.parse(handle, do_features)
  File "/Users/pc40583/repositories/biopython/Bio/GenBank/Scanner.py", line 473, in parse
    if self.feed(handle, consumer, do_features):
  File "/Users/pc40583/repositories/biopython/Bio/GenBank/Scanner.py", line 445, in feed
    self._feed_feature_table(consumer, self.parse_features(skip=False))
  File "/Users/pc40583/repositories/biopython/Bio/GenBank/Scanner.py", line 395, in _feed_feature_table
    consumer.location(location_string)
  File "/Users/pc40583/repositories/biopython/Bio/GenBank/__init__.py", line 1132, in location
    cur_feature.location = _loc(location_line, self._expected_size, strand)
  File "/Users/pc40583/repositories/biopython/Bio/GenBank/__init__.py", line 310, in _loc
    return SeqFeature.FeatureLocation(_pos(s, -1), _pos(e), strand, ref=ref)
  File "/Users/pc40583/repositories/biopython/Bio/SeqFeature.py", line 761, in __init__
    self.start))
ValueError: End location (159) must be greater than or equal to start location (<2643)
```

It would be reasonable and consistent to attempt to fix these malformed origin wrapping location in the same way (again, with a warning).

**Comment by nimne** [comment]
I'll take a look at this! I agree, it seems reasonable to 'fix' these in the same way as the defined features that span the origin.

**Comment by peterjc** [comment]
Fixed by #2162
